From 51c54dbecdc4c98bf7da0bf3c1c4aa6ac8c524c9 Mon Sep 17 00:00:00 2001
Message-Id: <51c54dbecdc4c98bf7da0bf3c1c4aa6ac8c524c9.1425610844.git.chang-joon.lee@intel.com>
In-Reply-To: <6fa2ab23adf6d7de781e906d04a0224410262d9b.1425610844.git.chang-joon.lee@intel.com>
References: <6fa2ab23adf6d7de781e906d04a0224410262d9b.1425610844.git.chang-joon.lee@intel.com>
From: Gaurav K Singh <gaurav.k.singh@intel.com>
Date: Fri, 13 Feb 2015 15:21:19 +0530
Subject: [PATCH 22/27] SQUASHME! [VPG]: video/adf/intel: Allow DSI PLL1 for
 DSI Dual link

SQUASHME! - This patch should be squashed into the following existing patch:
	Author: Ramalingam C <ramalingam.c@intel.com>
	Date:   Thu Feb 5 09:41:02 2015 +0530
	Change-Id: I253c48ae56fd139dff29cf47467f33493e16109f
	MUST_REBASE [VPG]: video/adf/intel: Implementation of VLV/CHV DSI DRRS

Issue: GMINL-5610
Change-Id: I4afac4d77e81eaa4c6e435057159bdbce808ad2f
Signed-off-by: Gaurav K Singh <gaurav.k.singh@intel.com>
---
 drivers/video/adf/intel/core/vlv/vlv_dsi_pll.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/video/adf/intel/core/vlv/vlv_dsi_pll.c b/drivers/video/adf/intel/core/vlv/vlv_dsi_pll.c
index 001464d..315c3f4 100644
--- a/drivers/video/adf/intel/core/vlv/vlv_dsi_pll.c
+++ b/drivers/video/adf/intel/core/vlv/vlv_dsi_pll.c
@@ -229,7 +229,7 @@ static void vlv_dsi_pll_reg_configure(struct vlv_pll *pll,
 			dsi_mnp->dsi_pll_ctrl |= DSI_PLL_CLK_GATE_DSI0_DSIPLL;
 
 		/* Enable DSI1 pll for DSI Port C & DSI Dual link*/
-		else if (dsi_ctx->ports & (1 << PORT_C))
+		if (dsi_ctx->ports & (1 << PORT_C))
 			dsi_mnp->dsi_pll_ctrl |= DSI_PLL_CLK_GATE_DSI1_DSIPLL;
 	} else {
 
-- 
1.7.9.5

